install_files/ansible-base/usr.sbin.apache2

```diff
@@ -206,6 +206,7 @@
   /var/www/securedrop/static/i/tipbox/tipbox-hed-submit3.png r,
   /var/www/securedrop/static/i/tipbox/tipbox-logo.png r,
   /var/www/securedrop/static/js/journalist.js r,
+  /var/www/securedrop/static/js/libs/jquery-2.1.4.js r,
```

Comment:
it will allow the user to load the source by changing the URL to end with **jquery-2.1.4.js** instead of the minified version which is the easiest way to comply. If the source is not available to the user who downloaded the minified version it gets more complicated.

That being said I'm not an apparmor expert and maybe there is no need for this line and the file is readable regardless ?